integration-cli/docker_cli_ps_test.go

```diff
@@ -787,3 +787,30 @@ func (s *DockerSuite) TestPsShowMounts(c *check.C) {
 	out, _ = dockerCmd(c, "ps", "--format", "{{.Names}} {{.Mounts}}", "--filter", "volume="+prefix+slash+"this-path-was-never-mounted")
 	c.Assert(strings.TrimSpace(string(out)), checker.HasLen, 0)
 }
+
+func (s *DockerSuite) TestPsListContainersFilterNetwork(c *check.C) {
+	// create a container
+	out, _ := dockerCmd(c, "run", "-d", "--net=bridge", "--name=onbridgenetwork", "busybox", "top")
+	out, _ = dockerCmd(c, "run", "-d", "--net=none", "--name=onnonenetwork", "busybox", "top")
```

Comment:
these should probably be 

    runSleepingContainer(c, "-d", ...., ...)

that's a utility method to start the right image, with the right command, depending on the platform (linux/windows)